src/toolkits/object_detection/object_detector.hpp

```diff
@@ -10,6 +10,7 @@
 #include <functional>
 #include <map>
 #include <memory>
+#include <sstream>
```

Comment:
Why are we adding a header to the `.hpp` file if it's not being used in the `.hpp` file?

Reply:
Moved it to the cpp file.